stdlib/public/core/Integers.swift.gyb

```diff
@@ -447,41 +504,37 @@ public func ${x.operator}= <
   lhs = lhs ${x.operator} rhs
 }
 
-@_transparent
+//@_transparent
 public func ${x.nonMaskingOperator} <
   T: FixedWidthInteger, U: BinaryInteger
 >(lhs: T, rhs: U) -> T {
-  // FIXME(integers): uncomment once Int conforms to BinaryInteger
-  //let shift = rhs < -T.bitWidth ? -T.bitWidth
-            //: rhs > T.bitWidth ? T.bitWidth
-            //: Int(rhs)
-  //return lhs ${x.nonMaskingOperator} shift
-  fatalError()
+  let shift = rhs < -T.bitWidth ? -T.bitWidth
+            : rhs > T.bitWidth ? T.bitWidth
+            : Int(rhs)
+  return lhs ${x.nonMaskingOperator} shift
 }
 
 //===----------------------------------------------------------------------===//
 //=== "Smart ${x.description}", supporting overshifts and negative shifts -===//
 //===----------------------------------------------------------------------===//
 
-@_transparent
+//@_transparent
 public func ${x.nonMaskingOperator} <
   T: FixedWidthInteger
 >(lhs: T, rhs: Int) -> T {
-  // FIXME(integers): uncomment once Int conforms to BinaryInteger
-  fatalError()
-  //let overshiftR = T.isSigned ? lhs &>> (T.bitWidth - 1) : 0
-  //let overshiftL: T = 0
-  //if _fastPath(rhs >= 0) {
-    //if _fastPath(rhs < T.bitWidth) {
-      //return lhs.${x.name}(T(extendingOrTruncating: rhs))
-    //}
-    //return overshift${'LR'['R' in x.name]}
-  //}
-
-  //if _slowPath(rhs <= -T.bitWidth) {
-    //return overshift${'RL'['R' in x.name]}
-  //}
-  //return lhs ${x.operator.translate(maketrans('<>', '><'))} -rhs
```

Comment:
It’s not exactly clear to me what the basis for this comparison is, or how to evaluate the changes.  Could you clarify?